overblog/graphiql-bundle/0.1/post-install.txt

```diff
@@ -0,0 +1,3 @@
+<bg=blue;fg=white> Next for OverblogGraphiQLBundle </>
+    1. The bundle is auto configured now, if you need special/custom configuration read the documentation <comment>https://github.com/overblog/GraphiQLBundle/blob/master/Resources/doc/definitions/index.md</>
```

Comment:
This first comment can be removed as we don't link to the docs in post install information. What you can do instead is add the link as a comment in the configuration file (see other recipes to use the same semantic).

Reply:
but there is no such config file (as the default settings are meaningful).

anyway, I don't think we need to link to the doc at all from Flex here (as you only need for more advanced cases)

Reply:
Exactly, if we put an empty file with:
`overblog_graphiql: ~`
the bot won't allow it

I've then removed the link, would still be good to point users to documentation somehow